README.md

```diff
@@ -43,6 +43,18 @@ To upgrade to the latest (unstable) version, run
 pip install --upgrade git+https://github.com/cornellius-gp/gpytorch.git
 ```
 
+### ArchLinux Package
```

Comment:
Change from `###` to `####` to match the previous header level

Reply:
Ah my bad.